From e661fab33f82017d2b0577889ea1de58e7023c40 Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Thu, 23 Apr 2020 12:25:35 +0200
Subject: [PATCH 1/1] Patching for Buildroot

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3ce963e..2532291 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,7 +26,7 @@ find_package(Qt5 ${QT_MIN_VERSION} CONFIG REQUIRED COMPONENTS
     Qml
 )
 
-find_package(LibKWorkspace 5.14.90 CONFIG REQUIRED)
+#find_package(LibKWorkspace 5.14.90 CONFIG REQUIRED)
 
 find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS I18n Plasma Package)
 
-- 
2.26.1

