From a38b8dd4b0cc5a5fb2f20a95a33cf9213c37b92d Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 8 Jul 2011 20:25:16 +0200
Subject: [PATCH 194/198] Add localversion for -RT release

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 localversion-rt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
new file mode 100644
index 000000000000..483ad771f201
--- /dev/null
+++ b/localversion-rt
@@ -0,0 +1 @@
+-rt19
-- 
2.44.0

