From aca62ce9812a4e6a11d1ae1cb1aac80f98c4343c Mon Sep 17 00:00:00 2001
From: Clark Williams <clark.williams@gmail.com>
Date: Tue, 25 Jun 2024 17:08:39 -0500
Subject: [PATCH 196/198] Linux 6.6.35-rt34 REBASE

Signed-off-by: Clark Williams <clark.williams@gmail.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..21988f9ad53f 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt34
-- 
2.45.2

