From f81d441361f48874bf60195d27353323d287fab5 Mon Sep 17 00:00:00 2001
From: Clark Williams <clark.williams@gmail.com>
Date: Mon, 8 Apr 2024 21:34:33 -0500
Subject: [PATCH 198/198] Linux 6.6.25-rt29 REBASE

Signed-off-by: Clark Williams <clark.williams@gmail.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..90290c642ed5 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt29
-- 
2.44.0

