From 28778f34d2000811bacc10287123e9b43ef5c259 Mon Sep 17 00:00:00 2001
From: Clark Williams <clark.williams@gmail.com>
Date: Sun, 21 Jul 2024 14:43:10 -0500
Subject: [PATCH 198/198] Linux 6.6.41-rt37 REBASE

Signed-off-by: Clark Williams <clark.williams@gmail.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9ad53f..a3b2408c1da6 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt37
-- 
2.45.2

