From bdf1875bcd4e34766d45a6514ec9fed9d2cf1a9a Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Wed, 8 Jun 2022 11:44:11 +0200
Subject: [PATCH 1/1] Try to use 4 threads Tensorflow v2.3.0+ supports
 multithreading.

---
 precise_lite_runner/runner.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/precise_lite_runner/runner.py b/precise_lite_runner/runner.py
index 7e56a30..0aadf73 100644
--- a/precise_lite_runner/runner.py
+++ b/precise_lite_runner/runner.py
@@ -16,6 +16,11 @@ class TFLiteRunner:
     def __init__(self, model_name: str):
         #  Setup tflite environment
         self.interpreter = tflite.Interpreter(model_path=model_name)
+        try:
+            self.interpreter.set_num_threads(4)
+        except:
+            pass
+
         self.interpreter.allocate_tensors()
 
         self.input_details = self.interpreter.get_input_details()
-- 
2.34.1

