From d49aa5b51128d0bf38fb902a57d1965e8983af5f Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Tue, 27 Jul 2021 15:01:43 +0200
Subject: [PATCH 1/1] Simplify setup

---
 setup.py | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/setup.py b/setup.py
index 1de4f3a..de629bd 100644
--- a/setup.py
+++ b/setup.py
@@ -18,25 +18,12 @@ __author__ = 'seanfitz'
 import os
 from setuptools import setup
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
-
-
-def required(requirements_file):
-    """Read requirements file and remove comments and empty lines."""
-    base_dir = os.path.abspath(os.path.dirname(__file__))
-    with open(os.path.join(base_dir, requirements_file), 'r') as f:
-        requirements = f.read().splitlines()
-        return [pkg for pkg in requirements
-                if pkg.strip() and not pkg.startswith("#")]
-
 setup(
     name="adapt-parser",
     version="0.5.1",
     author="Sean Fitzgerald",
     author_email="sean@fitzgeralds.me",
     description=("A text-to-intent parsing framework."),
-    long_description=long_description,
     long_description_content_type="text/markdown",
     license=("Apache License 2.0"),
     keywords="natural language processing",
@@ -54,6 +41,4 @@ setup(
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
     ],
-
-    install_requires=required('requirements.txt')
 )
-- 
2.20.1

