From b457885eba19f8be0be2da35f5dbfca423be651f Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Thu, 7 Jan 2021 17:12:04 +0100
Subject: [PATCH 1/1] Make python-pytz available as host package

---
 package/python-pytz/python-pytz.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/python-pytz/python-pytz.mk b/package/python-pytz/python-pytz.mk
index 888f3c4556..7a64656bef 100644
--- a/package/python-pytz/python-pytz.mk
+++ b/package/python-pytz/python-pytz.mk
@@ -12,3 +12,4 @@ PYTHON_PYTZ_LICENSE = MIT
 PYTHON_PYTZ_LICENSE_FILES = LICENSE.txt
 
 $(eval $(python-package))
+$(eval $(host-python-package))
-- 
2.20.1

