From 0d0a3cd42688a792b6845482fccf2985b12cf631 Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Mon, 28 Oct 2019 14:31:47 +0100
Subject: [PATCH 1/1] Install glibc locale binary into target

---
 package/glibc/glibc.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/glibc/glibc.mk b/package/glibc/glibc.mk
index 68c6670007..ba28b668b0 100644
--- a/package/glibc/glibc.mk
+++ b/package/glibc/glibc.mk
@@ -144,6 +144,8 @@ GLIBC_LIBS_LIB += libthread_db.so.*
 endif
 
 define GLIBC_INSTALL_TARGET_CMDS
+	$(INSTALL) -m 0755 -D $(@D)/build/locale/locale \
+		$(TARGET_DIR)/usr/bin/locale
 	for libpattern in $(GLIBC_LIBS_LIB); do \
 		$(call copy_toolchain_lib_root,$$libpattern) ; \
 	done
-- 
2.11.0

