From bb1a1b7434aaadfe3abd2154344fd17855e0ae52 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Mon, 1 Jun 2020 17:08:11 +0200
Subject: [PATCH 1/1] Bump QT5 to 5.15.0

---
 package/qt5/qt5.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/qt5/qt5.mk b/package/qt5/qt5.mk
index 28de0e5b70..e540ebcf85 100644
--- a/package/qt5/qt5.mk
+++ b/package/qt5/qt5.mk
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-QT5_VERSION_MAJOR = 5.12
-QT5_VERSION = $(QT5_VERSION_MAJOR).8
+QT5_VERSION_MAJOR = 5.15
+QT5_VERSION = $(QT5_VERSION_MAJOR).0
 QT5_SOURCE_TARBALL_PREFIX = everywhere-src
 QT5_SITE = https://download.qt.io/archive/qt/$(QT5_VERSION_MAJOR)/$(QT5_VERSION)/submodules
 
-- 
2.20.1

