From 198ab7d0e7cd1862d84b2cae029a4b2c6471d945 Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Sat, 25 Apr 2020 21:11:34 +0200
Subject: [PATCH 1/1] Upgrade KF5 to version 5.69

---
 package/kf5/kf5.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/kf5/kf5.mk b/package/kf5/kf5.mk
index 8360fc9b63..3386c14ca4 100644
--- a/package/kf5/kf5.mk
+++ b/package/kf5/kf5.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-KF5_VERSION_MAJOR = 5.47
+KF5_VERSION_MAJOR = 5.69
 KF5_VERSION = $(KF5_VERSION_MAJOR).0
 KF5_SITE = https://download.kde.org/stable/frameworks/$(KF5_VERSION_MAJOR)
 
-- 
2.26.1

