From 4e0dad71c4aedd6cc6319e3c9e5978cb0e4046d6 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Fri, 18 Mar 2022 19:19:41 +0100
Subject: [PATCH 1/1] pulse fix

---
 application/qml.qrc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/application/qml.qrc b/application/qml.qrc
index e71b72f..f869bb6 100644
--- a/application/qml.qrc
+++ b/application/qml.qrc
@@ -14,7 +14,6 @@
         <file alias="panel/quicksettings/ShutdownDelegate.qml">qml/panel/quicksettings/ShutdownDelegate.qml</file>
         <file alias="panel/quicksettings/SliderBase.qml">qml/panel/quicksettings/SliderBase.qml</file>
         <file alias="panel/quicksettings/VolumeSlider.qml">qml/panel/quicksettings/VolumeSlider.qml</file>
-        <file alias="panel/quicksettings/VolumeSliderPulseAudio.qml">qml/panel/quicksettings/VolumeSliderPulseAudio.qml</file>
         <file alias="panel/quicksettings/WirelessDelegate.qml">qml/panel/quicksettings/WirelessDelegate.qml</file>
         <file alias="panel/quicksettings/brightness-decrease.svg">qml/panel/quicksettings/brightness-decrease.svg</file>
         <file alias="panel/quicksettings/brightness-increase.svg">qml/panel/quicksettings/brightness-increase.svg</file>
-- 
2.20.1

