From 10eedaf25c738042836e4cddc525deb693cb0794 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Wed, 28 Oct 2020 10:27:10 +0100
Subject: [PATCH 1/1] No X11

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17eb825d..2c79cede 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,9 +49,9 @@ if(WIN32)
     find_package(KDEWin REQUIRED)
 endif()
 
-if (NOT APPLE AND NOT WIN32)
-    find_package(X11)
-endif()
+#if (NOT APPLE AND NOT WIN32)
+#    find_package(X11)
+#endif()
 set(HAVE_X11 ${X11_FOUND})
 if (HAVE_X11)
   add_feature_info("X11 Session Management (libSM)" X11_SM_FOUND "Support for session management in KApplication")
-- 
2.20.1

