From c33439e2093fe832effcdbe4f04fb912140a40e2 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Sun, 16 May 2021 11:36:09 +0200
Subject: [PATCH 1/1] Add entry_points for binaries

---
 setup.py | 15 ++++++++++++++-
 1 file changed, 14 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index d8b11261..96feec0f 100644
--- a/setup.py
+++ b/setup.py
@@ -87,5 +87,18 @@ setup(
 
     },
     packages=find_packages(include=['mycroft*']),
-    include_package_data=True
+    include_package_data=True,
+
+    entry_points={
+        'console_scripts': [
+            'mycroft-speech-client=mycroft.client.speech.__main__:main',
+            'mycroft-messagebus=mycroft.messagebus.service.__main__:main',
+            'mycroft-skills=mycroft.skills.__main__:main',
+            'mycroft-audio=mycroft.audio.__main__:main',
+            'mycroft-echo-observer=mycroft.messagebus.client.ws:echo',
+            'mycroft-audio-test=mycroft.util.audio_test:main',
+            'mycroft-enclosure-client=mycroft.client.enclosure.__main__:main',
+            'mycroft-cli-client=mycroft.client.text.__main__:main'
+        ]
+    }
 )
-- 
2.20.1

