From 862ebe4c08b2b64d7baab17dd1f8ce044c5fda90 Mon Sep 17 00:00:00 2001
From: Aditya Mehra <aix.m@outlook.com>
Date: Sat, 14 Aug 2021 12:53:49 +0930
Subject: [PATCH] fix property name

---
 ui/alarm.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/alarm.qml b/ui/alarm.qml
index 01ac372..07431df 100644
--- a/ui/alarm.qml
+++ b/ui/alarm.qml
@@ -9,7 +9,7 @@ Mycroft.CardDelegate {
     id: root
     property bool alarmExpired: sessionData.alarmExpired
     property color alarmColor: alarmExpired ? "#FFFFFF" : "#22A7F0"
-    cardBackgoundOverlayColor: "black"
+    cardBackgroundOverlayColor: "black"
     
     Rectangle {
         id: leftRect
