From c33880b2b4561bc3dec61e6328debcbddba9a7f1 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Wed, 6 Jan 2021 20:06:43 +0100
Subject: [PATCH 1/1] Shut up

---
 mycroft/skills/skill_updater.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mycroft/skills/skill_updater.py b/mycroft/skills/skill_updater.py
index ace247e2db..3441de007e 100644
--- a/mycroft/skills/skill_updater.py
+++ b/mycroft/skills/skill_updater.py
@@ -118,7 +118,7 @@ class SkillUpdater:
         platform_default_skills = default_skill_groups.get(self.msm.platform)
         if platform_default_skills is None:
             log_msg = 'No default skills found for platform {}'
-            LOG.info(log_msg.format(self.msm.platform))
+            #LOG.info(log_msg.format(self.msm.platform))
         else:
             default_skills.update(platform_default_skills)
 
-- 
2.20.1

