From 28a9ff65a9b71ad70b906d688b7b2f593ea108cc Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Fri, 5 Oct 2018 09:42:55 +0200
Subject: [PATCH 1/1] Enforce UTF8 encoding

---
 mycroft/dialog/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mycroft/dialog/__init__.py b/mycroft/dialog/__init__.py
index 59b4a44a0d..59c521b929 100644
--- a/mycroft/dialog/__init__.py
+++ b/mycroft/dialog/__init__.py
@@ -45,7 +45,7 @@ class MustacheDialogRenderer(object):
             template_name (str): a unique identifier for a group of templates
             filename (str): a fully qualified filename of a mustache template.
         """
-        with open(filename, 'r') as f:
+        with open(filename, 'r', encoding='utf-8') as f:
             for line in f:
                 template_text = line.strip()
                 if template_name not in self.templates:
-- 
2.11.0

