From 7acffbe16b094228cb695370518ad658818a8348 Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Fri, 3 Jan 2020 10:19:38 +0100
Subject: [PATCH 1/1] Don't bother the executable rights

---
 setup.py | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/setup.py b/setup.py
index acb0d04..97b3769 100644
--- a/setup.py
+++ b/setup.py
@@ -15,27 +15,10 @@ if sys.version_info < (2, 6):
     sys.exit(1)
 
 
-FILES_TO_MARK_EXECUTABLE = ["flac-linux-x86", "flac-linux-x86_64", "flac-mac", "flac-win32.exe"]
-
-
 class InstallWithExtraSteps(install):
     def run(self):
         install.run(self)  # do the original install steps
 
-        # mark the FLAC executables as executable by all users (this fixes occasional issues when file permissions get messed up)
-        for output_path in self.get_outputs():
-            if os.path.basename(output_path) in FILES_TO_MARK_EXECUTABLE:
-                log.info("setting executable permissions on {}".format(output_path))
-                stat_info = os.stat(output_path)
-                os.chmod(
-                    output_path,
-                    stat_info.st_mode |
-                    stat.S_IRUSR | stat.S_IXUSR |  # owner can read/execute
-                    stat.S_IRGRP | stat.S_IXGRP |  # group can read/execute
-                    stat.S_IROTH | stat.S_IXOTH  # everyone else can read/execute
-                )
-
-
 setup(
     name="SpeechRecognition",
     version=speech_recognition.__version__,
-- 
2.24.0.rc1

