From 9116b1bcc3a1230015c70af9116e4abb51f2d8ce Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Tue, 27 Oct 2020 15:02:48 +0100
Subject: [PATCH 1/1] No X11

---
 CMakeLists.txt | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f08d6d5..e8629c2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,15 +47,15 @@ find_package(KF5DBusAddons ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5WindowSystem ${KF5_DEP_VERSION} REQUIRED)
 
 # no X11 stuff on mac
-if (NOT APPLE)
-    find_package(XCB MODULE COMPONENTS XCB KEYSYMS XTEST XKB)
-    set_package_properties(XCB PROPERTIES DESCRIPTION "X protocol C-language Binding"
-                       URL "http://xcb.freedesktop.org"
-                       TYPE OPTIONAL
-                      )
-
-    find_package(X11)
-endif()
+#if (NOT APPLE)
+#    find_package(XCB MODULE COMPONENTS XCB KEYSYMS XTEST XKB)
+#    set_package_properties(XCB PROPERTIES DESCRIPTION "X protocol C-language Binding"
+#                       URL "http://xcb.freedesktop.org"
+#                       TYPE OPTIONAL
+#                      )
+#
+#    find_package(X11)
+#endif()
 
 set(HAVE_X11 0)
 
-- 
2.20.1

