From 6daeedb4399a8129a2748efdc3ac97bf61b2db5d Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Fri, 4 Sep 2020 14:14:58 +0200
Subject: [PATCH 1/1] Set codec_dai clockspeed to 24000000

---
 seeed-voicecard.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/seeed-voicecard.c b/seeed-voicecard.c
index 1ed9a0e..630939b 100644
--- a/seeed-voicecard.c
+++ b/seeed-voicecard.c
@@ -457,6 +457,7 @@ static int seeed_voice_card_dai_link_of(struct device_node *node,
 	#else
 	ret = asoc_simple_parse_clk_codec(dev, codec, dai_link, codec_dai);
 	#endif
+	codec_dai->sysclk = 24000000;
 	if (ret < 0)
 		goto dai_link_of_err;
 
-- 
2.20.1

