From ec57a8c3ff7fbbd78e7587403ee60ca7ae18dce9 Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Fri, 24 Apr 2020 02:16:46 +0200
Subject: [PATCH 1/1] Remove docbook requirement

---
 CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index abaebec..4a5743b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,19 +63,6 @@ if (NOT LIBXML2_XMLLINT_EXECUTABLE)
    message(FATAL_ERROR "xmllint is required to process DocBook XML")
 endif()
 
-find_package(DocBookXML4 "4.5")
-
-set_package_properties(DocBookXML4 PROPERTIES
-                       TYPE REQUIRED
-                       PURPOSE "Required by the KDE help system to process DocBook XML"
-                      )
-
-find_package(DocBookXSL)
-set_package_properties(DocBookXSL PROPERTIES
-                       TYPE REQUIRED
-                       PURPOSE "Required by the KDE help system to process DocBook XML"
-                      )
-
 remove_definitions(-DQT_NO_CAST_FROM_ASCII)
 
 set(_kdoctoolsBootStrapping TRUE)
-- 
2.26.1

