From 32515bb6e2427c5792ee45173c9af52293cdfc25 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Wed, 28 Oct 2020 11:25:12 +0100
Subject: [PATCH 1/1] No X11

---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 447ff81..c8e4760 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,10 +37,10 @@ if(APPLE)
     cmake_find_frameworks(CoreFoundation Carbon)
 endif()
 
-if(NOT APPLE)
-    find_package(X11)
-    find_package(XCB COMPONENTS XCB)
-endif()
+#if(NOT APPLE)
+#    find_package(X11)
+#    find_package(XCB COMPONENTS XCB)
+#endif()
 
 if(X11_FOUND)
     find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED X11Extras)
-- 
2.20.1

