From 28d7e2feb6e5579a6cd8f2e0de51e7b05bb78095 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Tue, 10 May 2022 09:13:31 +0200
Subject: [PATCH 1/1] Set log level to warning

---
 src/filelock/_api.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/filelock/_api.py b/src/filelock/_api.py
index 382f5cb..6eb6d1e 100644
--- a/src/filelock/_api.py
+++ b/src/filelock/_api.py
@@ -8,7 +8,7 @@ from typing import Optional, Type, Union
 from ._error import Timeout
 
 _LOGGER = logging.getLogger("filelock")
-_LOGGER.setLevel(logging.DEBUG)
+_LOGGER.setLevel(logging.WARNING)
 
 
 # This is a helper class which is returned by :meth:`BaseFileLock.acquire` and wraps the lock to make sure __enter__
-- 
2.34.1

