From 8ceb98d315404a1523974b7b2c2e1370b5ef61cd Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Sat, 10 Oct 2020 19:49:51 +0200
Subject: [PATCH 1/1] Remove KDoctools requirement

---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eb6d5a8..8d81e02 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,6 @@ find_package(Qt5 ${QT_MIN_VERSION} REQUIRED COMPONENTS
 find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
     CoreAddons
     Declarative
-    DocTools
     GlobalAccel
     Notifications
     I18n
@@ -79,7 +78,6 @@ configure_file(config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config.h)
 add_subdirectory(applet)
 add_subdirectory(src)
 add_subdirectory(data)
-add_subdirectory(doc)
 
 # add clang-format target for all our real source files
 file(GLOB_RECURSE ALL_CLANG_FORMAT_SOURCE_FILES *.cpp *.h)
-- 
2.20.1

